prow/plugins/updateconfig/README.md

```diff
@@ -17,17 +17,14 @@ config_updater:
     # Update the thing-config configmap whenever thing changes
     path/to/some/other/thing:
       name: thing-config
-      # Using ProwJobNamespace by default.
+    # Specify the clusters and namespaces
```

Comment:
Why is this comment here when there is no cluster and namespace specification afterwards?
Please instead mention that if cluster and namespace configuration are unset, it will be put into the `default` cluster in the `prowjob` namespace

Reply:
> @alvaroaleman We set up clusters and namespaces here.

Yes but you are removing the information that this is getting defaulted and the `cluster and namespaces` are optional config options: https://github.com/kubernetes/test-infra/blob/b2cf66401ce642c5e726fdb5cfd1d43eb67a88be/prow/plugins/config.go#L844-L846

Reply:
Replaced. Thanks!